### PR TITLE
Uptime monitor status alert 

### PR DESCRIPTION
**Pull request by maclin-masterson**
Add uptime monitor status alert type to kibana_alert module. 

This PR will add new parameters to the kibana_alert module, as well as new functions in module_utils/kibana.py to better handle the differences between alert types when calling ensure_alert()

### CODE REVIEW

**Review by ianrscottexp — APPROVED** [review]

**Review by jesselarrabee — APPROVED** [review]

**Review by mgaruccio — APPROVED** [review]
ltgm

File: plugins/module_utils/kibana.py [hunk 1]
```diff
@@ -96,21 +96,14 @@ def ensure_alert(self, alert_id=None):
       # Creating an alert
       method="POST"
 
-    criteria = self.format_alert_conditions()
-
```
[the remark this answers is not in the document]
Comment (maclin-masterson) [reply plugins/module_utils/kibana.py:100]: This got moved into the format_alert_params() function. This line is only needed for metrics_threshold alerts, so it is placed inside of an if statement to check for the alert type.

https://github.com/Expedient/ansible-collection-elastic/blob/uptime_monitor_status_alert_rev2/plugins/module_utils/kibana.py#L177

File: plugins/module_utils/kibana.py [hunk 2]
```diff
@@ -96,21 +96,14 @@ def ensure_alert(self, alert_id=None):
...
     # set variables for data
     notify_when = self.module.params.get('notify_on')
-    alert_on_no_data = self.module.params.get('alert_on_no_data')
```
[the remark this answers is not in the document]
Comment (maclin-masterson) [reply plugins/module_utils/kibana.py:103]: Yep, still supported. alert_on_no_data is specific to metrics_threshold alerts, so it has been moved to `format_alert_params()`

**Comment by maclin-masterson** [comment]
PR has been approved. Merging.

**Pull request merged**